tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-local/src/main/java/org/springframework/cloud/stream/app/task/launcher/local/sink/TaskLauncherLocalSinkConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.task.launcher.local.sink;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.messaging.Sink;
+import org.springframework.cloud.task.launcher.annotation.EnableTaskLauncher;
+
+/**
+ * Configuration class for the TaskLauncherSink.
+ *
+ * @author Glenn Renfro
+ */
+@EnableBinding(Sink.class)
+@EnableConfigurationProperties(TaskLauncherLocalSinkProperties.class)
```

Comment:
Forgive my ignorance, but is this needed if the class is going to be empty (`TaskLauncherLocalSinkProperties`)?